BlueprintUICommonControls/Tests/Sources/AttributedLabelTests.swift

```diff
@@ -36,13 +36,21 @@ class AttributedLabelTests: XCTestCase {
         defaultTraits.accessBackingView(in: view) { label in
             let label = label as! UILabel
 
-            XCTAssertEqual(label.accessibilityTraits, [])
+            XCTAssertEqual(label.accessibilityTraits, [.none])
+
+            // The label actually has a static text trait, but not when the tests run outside an application host.
+            // UI-8266
```

Comment:
Fwiw https://github.com/squareup/market/blob/main/ios/Project.swift should show how to set up a test host app, IIRC

Reply:
Well this is frustrating. 

On my local machine, these tests pass while on CI they're failing. 
I've confirmed that im using the same `tuist` invocation to launch the tests 
I've confirmed that the tests are running in an active app host.

Finally, I've added a dump of the view hierarchy to the test results and compared my machine against the CI and found them to be are identical (modulo view frames and memory addresses)
<img width="1385" alt="Screenshot 2025-02-20 at 13 59 30" src="https://github.com/user-attachments/assets/7cce0030-6b0c-43ba-94f8-e1dd6c841823" />

Reply:
OK Ive got it working properly inside a UITest. which I've had to add as a third test target.